docs/content/en/docs/guidelines-for-contributing.md

```diff
@@ -0,0 +1,35 @@
+## Guidelines for contributing
```

Comment:
Again, this file is in the wrong directory -- see comments in your PR for github.  And that PR also includes the guidelines...  So need to straighten this all out.